testManager.py

```diff
@@ -65,14 +65,17 @@ async def sendRequest(session, url):
     try:
         #Fetch individual request content
         async with session.get(url) as response:
-            if response.status >= 200 and response.status <= 299:
+            if response.status < 200 or response.status > 299:
+
+                print(response.status)
```

Comment:
maybe remove test print? I don't think this will actually hit the shell, will it?